packages/data-broker/src/services/kobo/KoBoTranslator.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import keyBy from 'lodash.keyby';
+
+export class KoBoTranslator {
+  constructor(models) {
+    this.models = models;
+  }
+
+  async fetchEntityInfoFromKoBoAnswer(koboEntityCode) {
+    const entity = await this.models.entity.findOne({ code: `LA_sch_${koboEntityCode}` });
```

Comment:
I'm going to make this change against the other PR #2942 just because it will require adding a migration for the `data_service_entity` table and I don't want to resolve older branch with newer migration annoyance